src/main/java/Controller.java

```diff
@@ -0,0 +1,98 @@
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Controller {
+    public String hello = "Hello! I'm Duke :P\nWhat can I do for you?";
```

Comment:
Will these Strings change as the program runs? If not, consider making them final and changing the variable names to uppercase.